src/main/groovy/pl/allegro/tech/build/axion/release/VerifyReleaseTask.groovy

```diff
@@ -25,7 +29,7 @@ class VerifyReleaseTask extends DefaultTask {
 
         if (resolver.checkUncommittedChanges()) {
             boolean uncommittedChanges = repository.checkUncommittedChanges()
-            project.logger.quiet("Looking for uncommitted changes.. ${uncommittedChanges ? 'FAILED' : ''}")
+            logger.trace("Looking for uncommitted changes.. ${uncommittedChanges ? 'FAILED' : ''}")
```

Comment:
That's BTW a big question here - there is no way to map quiet level to one of slf4j's levels. You cannot use warn or error because these messages are neither former nor latter. At the same time you cannot use info since there is no guarantee that it'll not be filtered out by logging configuration.